independent-projects/bootstrap/classloader-commons/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-bootstrap-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-classloader-commons</artifactId>
```

Comment:
TBH I would just go with `quarkus-classloader`.

Reply:
I can do that if you prefer - but it doesn't have any actual Classloader ?  I'd find it a bit confusing,  if I didn't know the history.

Reply:
I guess it's not that important. I pictured the primary feature of a shared module to be a base `ClassLoader` class, but this is also fine.